CHANGELOG.md

```diff
@@ -7,20 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### New Features
 
+1. Added a `--coverage <true/false>` flag which can be used to run mosdepth in normal mode instead of fast-mode. This flag will also make sure all output from mosdepth is present in the output directory for each individual
```

Comment:
I'm not a fan of the naming, it's not really clear what it does from sight.
Perhaps `coverage_fast` or something?